### PR TITLE
Fix: #14499 Added alt text to images for Userguide: applications_auth.rst

### PR DESCRIPTION
**Pull request by maskboyAvi**
##### SUMMARY

Added alt text to images for Userguide: applications_auth.rst

related #14499
##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Docs

##### AWX VERSION

Latest

##### ADDITIONAL INFORMATION

Please add Hacktoberfest label to the PR . Thank You .

### CODE REVIEW

**Review by tvo318 — APPROVED** [review]
Looks good to me, thanks for doing this, @maskboyAvi 👍🏼

File: docs/docsite/rst/userguide/applications_auth.rst [hunk 1]
```diff
@@ -31,11 +31,11 @@ Access the Applications page by clicking **Applications** from the left navigati
 |Applications - home with example apps|
 
 .. |Applications - home with example apps| image:: ../common/images/apps-list-view-examples.png
-
+   :alt: Applications - home with example apps
```
Comment (oraNod) [inline docs/docsite/rst/userguide/applications_auth.rst]:
```suggestion
   :alt: Applications list view
```

File: docs/docsite/rst/userguide/applications_auth.rst [hunk 2]
```diff
@@ -31,11 +31,11 @@ Access the Applications page by clicking **Applications** from the left navigati
...
 If no other applications exist, only a gray box with a message to add applications displays.
 
 .. image:: ../common/images/apps-list-view-empty.png
-
+   :alt: Empty Applications List View
```
Comment (oraNod) [inline docs/docsite/rst/userguide/applications_auth.rst]:
```suggestion
   :alt: No applications found in the list view
```

File: docs/docsite/rst/userguide/applications_auth.rst [hunk 3]
```diff
@@ -59,6 +59,7 @@ The New Application window opens.
 |Create application|
 
 .. |Create application| image:: ../common/images/apps-create-new.png
+   :alt: Create New Application Window
```
Comment (oraNod) [inline docs/docsite/rst/userguide/applications_auth.rst]:
```suggestion
   :alt: Create new application dialog
```

File: docs/docsite/rst/userguide/applications_auth.rst [hunk 4]
```diff
@@ -72,7 +73,7 @@ The New Application window opens.
 4. When done, click **Save** or **Cancel** to abandon your changes. Upon saving, the client ID displays in a pop-up window.
 
 .. image:: ../common/images/apps-client-id-popup.png
-
+   :alt: Client Id pop-up Window
```
Comment (oraNod) [inline docs/docsite/rst/userguide/applications_auth.rst]:
```suggestion
   :alt: Client ID popup
```

File: docs/docsite/rst/userguide/applications_auth.rst [hunk 5]
```diff
@@ -86,7 +87,7 @@ Selecting the **Tokens** view displays a list of the users that have tokens to a
 |Applications - tokens list|
 
 .. |Applications - tokens list| image:: ../common/images/apps-tokens-list-view-examples.png
-
+   :alt: Applications - Tokens List View
```
Comment (oraNod) [inline docs/docsite/rst/userguide/applications_auth.rst]:
```suggestion
   :alt: Application tokens list view
```

File: docs/docsite/rst/userguide/applications_auth.rst [hunk 6]
```diff
@@ -108,3 +109,4 @@ Tokens are added through the Users screen and can be associated with an applicat
 To verify the application in the example above now shows the user with the appropriate token, go to the **Tokens** tab of the Applications window:
 
 .. image:: ../common/images/apps-tokens-list-view-example2.png
+   :alt: Verifying User's Token in Applications Window
```
Comment (oraNod) [inline docs/docsite/rst/userguide/applications_auth.rst]:
```suggestion
   :alt: Verifying a specific user application token
```

**Review by oraNod — COMMENTED** [review]
Thanks for the changes @maskboyAvi These are very helpful. I hope you don't mind but I've added a few suggestions to make these consistent and a little more meaningful for users with assistive technology.

Please feel free to just take the suggestions and squash without including any attribution in the commit.

**Review by oraNod — APPROVED** [review]
Thanks @maskboyAvi Let's merge this after the ci checks run.

**Pull request merged**